.github/workflows/rerun_workflow_on_push.yml

```diff
@@ -6,7 +6,7 @@ on:
     branches:
       - main
       - candlepin-*-HOTFIX
-      - feature/autoreg_v2
+      - feature/*
```

Comment:
you would also need to change the `Rerun workflows` step below, because it references `feature/autoreg_v2` specifically

Reply:
done